deploy/dataflow/setup.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+import subprocess
+from distutils.command.build import build as _build  # type: ignore
+
+import setuptools
+
+# ------------------------------------------------------------------------------------------------------------
+# TODO: replace with the actual service name you wish to use
+SERVICE_NAME = "<SERVICE NAME>"
+# TODO: replace with a token you got from the performance studio (https://profiler.granulate.io/installation)
+GPROFILER_TOKEN = "<TOKEN>"
+# ------------------------------------------------------------------------------------------------------------
+
+KEEP_LOGS = True
+
+
+class build(_build):
+    sub_commands = _build.sub_commands + [("ProfilerInstallationCommands", None)]
+
+
+COMMANDS = [
+    "wget -q https://github.com/Granulate/gprofiler/releases/latest/download/gprofiler -O /tmp/gprofiler",
+    "sudo chmod +x /tmp/gprofiler",
+    f"sudo nohup setsid /tmp/gprofiler -cu --token {GPROFILER_TOKEN} --service-name {SERVICE_NAME}"
+    f" --disable-pidns-check > {'/tmp/gprofiler.log' if KEEP_LOGS else '/dev/null'} 2>&1 &",
```

Comment:
You use here `nohup`, `setsid` and `&`, Are all of them really required?

Reply:
Removed `nohup` as it's redundant.
the `&` is required, otherwise it hangs on the command

Reply:
`nohup` is still here..

Reply:
Whoops, fixed